### ISSUE TITLE
Internal Server Error when preferred language isn't English or French

### ISSUE THREAD

**Issue by amongonz** [ocaml/ocaml.org#205]
When the `Accept-Language` header gives the most weight to any language other than English (`en`) or French (`fr`), v3 responds with 500 Internal Server Error.

I noticed that other users were able to access v3 while I hadn't been able to check it out for a while. After some testing I found out that the trigger was my browsers' `es-ES,es` preference. This happens even though `en-US,en` are listed with lower weight, in which case content negotiation should select English (or French if `fr` had any weight above it).

**Comment by tmattio** [comment]
@debugnik I can't seem to reproduce that on `main`. Would you mind testing again and if you reproduce copy a curl request to test here?

**Comment by amongonz** [comment]
I can't reproduce it on main and I also can't reproduce it on the live site anymore either. This issue must have been conveniently fixed by some commit deployed since I opened it, so it can be closed as far as I'm concerned.

For reference, bisecting the history it appears that this error was fixed by 501db5a74695d27b249ad425c694d97db8a32391. My repro was just:

    curl -fLH "Accept-Language: es,en;q=0.5" http://localhost:8080/

Finally, while testing I also noticed that main returns 500 for malformed `Accept-Language` strings (I accidentally tested a typo) rather than 400, but that's a separate matter.
